### PR TITLE
corrected mismatching of values between dashboard and CLI

### LINKED ISSUE #726: Default perf test values mismatch between dashboard and CLI
**Description**
The default perf test duration in the dashboard is `30s`, while in the `mesherycli` is set to 10 seconds. Similarly for QPS - 0 in the UI and 1 in the CLI,  and the concurrent requests (0 vs 1). Also, the flag name in the CLI is `parallel-requests`, while in the UI it's called `Concurrent requests` - it would be nice to be consistent.

**Expected Behavior**
Consistent values/naming between the UI and CLI.

### PR DESCRIPTION
**Pull request by ashiscs**
fixes #726 
Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

File: mesheryctl/cmd/perf.go [hunk 1]
```diff
@@ -127,9 +127,9 @@ func init() {
 	perfCmd.Flags().StringVar(&testName, "name", StringWithCharset(8), "DESCRIPTION")
 	perfCmd.Flags().StringVar(&testMesh, "mesh", "", "DESCRIPTION")
 	// perfCmd.Flags().StringVar(&testFile, "file", "", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&qps, "qps", "1", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&parallelRequests, "parallel-requests", "1", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testDuration, "duration", "10s", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&qps, "qps", "0", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&parallelRequests, "Concurrent requests", "1", "DESCRIPTION")
```
Comment (peterj) [inline mesheryctl/cmd/perf.go]: shouldn't this be `concurrent-requests`? I'd also suggest you rename the var name to actually match (e.g. `concurrentRequests`)

Comment (leecalcote) [reply mesheryctl/cmd/perf.go]: Great feedback, @peterj.

Thanks for taking this on, @ashiscs!

**Comment by ashiscs** [comment]
@peterj , @leecalcote sir any further changes is required??

**Comment by ashiscs** [comment]
@peterj , @leecalcote sir now it is good to be merged as the checks are not failing!!

**Pull request merged**